src/components/workflowEditor/nodes/CINode.tsx

```diff
@@ -1,70 +1,110 @@
+    isLinkedCI: boolean
+    isExternalCI: boolean
+    isTrigger: boolean
+    linkedCount: number
+    downstreams: NodeAttr[]
+    to: string
+    toggleCDMenu: () => void
+    configDiffView?: boolean
 }
 
 export class CINode extends Component<CINodeProps> {
-
-    renderCardContent() {
-        let classes = 'workflow-node cursor';
-        let pipeline = this.props.isLinkedCI ? "Build: Linked" : this.props.isExternalCI ? "Build: External" : "Build";
-        return <div className={classes}>
-            <Link to={this.props.to} className="dc__no-decor">
-                {this.props.linkedCount ? <Tippy className="default-tt" arrow={true} placement="bottom" content={this.props.linkedCount}>
-                    <span className="link-count"> <img src={link} className="icon-dim-12 mr-5" alt="" />
-                        {this.props.linkedCount}
-                    </span>
-                </Tippy> : null}
-                <div className="workflow-node__trigger-type workflow-node__trigger-type--create">{this.props.triggerType}</div>
+    renderReadOnlyCard() {
+        return (
+            <div className="workflow-node">
                 <div className="workflow-node__title flex">
                     <div className="workflow-node__full-width-minus-Icon">
-                        <span className="workflow-node__text-light">{pipeline}</span>
-                        <Tippy className="default-tt" arrow={true} placement="bottom" content={this.props.title}>
-                            <div className="dc__ellipsis-left">{this.props.title}</div>
-                        </Tippy>
+                        <span className="workflow-node__text-light">
+                            {this.props.isExternalCI ? 'Build: External' : 'Build'}
+                        </span>
+                        <div className="dc__ellipsis-left">{this.props.title}</div>
                     </div>
                     <div className="workflow-node__icon-common workflow-node__CI-icon"></div>
                 </div>
-            </Link>
-            <button className="workflow-node__add-cd-btn">
-                <Tippy className="default-tt" arrow={false} placement="top" content={
-                    <span style={{ display: "block", width: "145px" }}> Add deployment pipeline </span>}>
-                    <Add className="icon-dim-18 fcb-5" onClick={(event: any) => {
-                        event.stopPropagation();
-                        let { top, left } = event.target.getBoundingClientRect();
-                        top = top + 25;
-                        this.props.toggleCDMenu();
-                    }} />
-                </Tippy>
-            </button>
-        </div>
+            </div>
+        )
+    }
+
+    renderCardContent() {
+        let classes = 'workflow-node cursor'
+        let pipeline = this.props.isLinkedCI ? 'Build: Linked' : this.props.isExternalCI ? 'Build: External' : 'Build'
+        return (
+            <div className={classes}>
```

Comment:
Why aren't we using classes directly?

Reply:
Not sure. Old implementation 😃 
Changed it to use directly.